presto-hive/src/test/java/com/facebook/presto/hive/TestHiveFileFormats.java

```diff
@@ -277,6 +278,40 @@ public boolean apply(TestColumn testColumn)
     }
 
     @Test
+    public void testParquetUseColumnNames()
+            throws Exception
+    {
+        List<TestColumn> testColumns =
+                TEST_COLUMNS.stream()
+                        .filter(column -> !ImmutableSet.of("t_complex", "t_array_empty", "t_map_null_key")
+                                .contains(column.getName()))
+                        .filter(column -> !hasType(
+                                column.getObjectInspector(),
+                                PrimitiveCategory.DATE,
+                                PrimitiveCategory.BINARY))
+                        .collect(Collectors.toList());
+
+        HiveOutputFormat<?, ?> outputFormat = new MapredParquetOutputFormat();
+        InputFormat<?, ?> inputFormat = new MapredParquetInputFormat();
+        @SuppressWarnings("deprecation")
+        SerDe serde = new ParquetHiveSerDe();
+        File file = File.createTempFile("presto_test", "parquet");
+        file.delete();
+        try {
+            // Reverse the order of the columns to test access by name, not by index
+            Collections.reverse(testColumns);
```

Comment:
hm. I suppose that's ok, it'll just have slightly different results than all of the other tests